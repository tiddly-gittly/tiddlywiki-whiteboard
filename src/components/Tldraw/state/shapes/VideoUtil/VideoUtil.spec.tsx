import { Video } from '..'

describe('Video shape', () => {
  it('Creates a shape', () => {
    expect(Video.create({ id: 'video' })).toMatchSnapshot('video')
  })
})
