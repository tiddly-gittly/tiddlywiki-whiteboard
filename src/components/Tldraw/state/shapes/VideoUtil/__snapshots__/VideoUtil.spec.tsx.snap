// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video shape Creates a shape: video 1`] = `
Object {
  "assetId": "assetId",
  "childIndex": 1,
  "currentTime": 0,
  "id": "video",
  "isPlaying": true,
  "name": "Video",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    1,
    1,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": false,
    "scale": 1,
    "size": "small",
  },
  "type": "video",
}
`;
